### DISCUSSION TITLE
Skin collections

### DISCUSSION THREAD

**Discussion by IdkeMint** [ppy/osu#21687] [Ideas]
make collections for skins ( for me it's really useful because sometimes i can't find skins that i use often and this thing can make this process not that boring)
![Screenshot 2022-12-17 225041](https://user-images.githubusercontent.com/100031245/208265800-e5ffcca4-ea2b-479d-93c4-c8dafd2f9118.png)

**Comment by frenzibyte** [comment]
There'll be support for dropdown searching which may be enough to make finding your skins easier (see https://github.com/ppy/osu/discussions/17020)

**Reply by IdkeMint** [reply]
collections is not just search it's sorting by groups so you can sort by mods, by your favorites etc
